### PR TITLE
FIX contacts disabled are not returned on create invoice

### PR DESCRIPTION
**Pull request by atm-GregM**
FIX contacts disabled are not returned on create invoice

when creating an invoice, the contact is added if he's not in disabled status

### CODE REVIEW

**Comment by eldy** [comment]
Can you provide a screenshot of the page that report wrong content before the fix ?

**Comment by atm-GregM** [comment]
Currently if I create an invoice, a contact is attached even if it is disabled ( contact).
Our proposal is not to attach the contact if it is.
Cordially

Test create invoice with contact disabled : 
image

**Comment by atm-maxime** [comment]
When creating a document (proposal, invoice, ...) contacts are automatically added to the document regarding their roles, even if the contact is disabled. The goal of this PR is to prevent adding disabled contacts to new documents.

File: htdocs/contact/class/contact.class.php [hunk 1]
```diff
@@ -1776,10 +1776,13 @@ public function getContactRoles($element = '')
 		$sql = "SELECT sc.fk_socpeople as id, sc.fk_c_type_contact";
 		$sql .= " FROM ".MAIN_DB_PREFIX."c_type_contact tc";
 		$sql .= ", ".MAIN_DB_PREFIX."societe_contacts sc";
+		$sql .= " INNER JOIN ".MAIN_DB_PREFIX."socpeople sp";
+		$sql .= " ON sc.fk_socpeople = sp.rowid";
 		$sql .= " WHERE sc.fk_soc =".((int) $this->socid);
 		$sql .= " AND sc.fk_c_type_contact=tc.rowid";
 		$sql .= " AND tc.element = '".$this->db->escape($element)."'";
 		$sql .= " AND tc.active = 1";
+		$sql .= " AND sp.statut = 1";
```
Comment (eldy) [inline htdocs/contact/class/contact.class.php:1785]: When you use the INNER JOIN syntax, you must and the condition on the same line than the "ON ..." so on line 1780
$sql .= " ON sc.fk_socpeople = sp.rowid AND sp.statut = 1";

Comment (atm-GregM) [reply htdocs/contact/class/contact.class.php]: Update OK

**Comment by atm-maxime** [comment]
@eldy is this ok for you ?
And why changing the base to 16 ? This fixes an issue in the 15.0 version.

**Pull request merged**